contacts/tests.py

```diff
@@ -1,3 +1,125 @@
-# from django.test import TestCase
+import pytest
+from users.models import User, Seeker
+from apartments.models import Apartment, City
+from .models import Connection
+from django.db import transaction, IntegrityError
 
-# Create your tests here.
+
+@pytest.fixture
+def new_city():
+    city = City(cityName='Tel Aviv')
+    city.save()
+    return city
+
+
+@pytest.fixture
+def fixture_make_seeker(new_city):
+    def make_seeker(user, start_date, min_rent, max_rent, roomates, rooms, about):
```

Comment:
_make_seeker*

Reply:
I renamed the inner function '_make_seeker' as you suggested